src/pretix/presale/templates/pretixpresale/event/fragment_addon_choice.html

```diff
@@ -285,10 +285,7 @@ <h4 id="cp-{{ form.pos.pk }}-item-{{ item.pk }}-legend">{{ item.name }}</h4>
                             {% if c.max_count == 1 or not c.multi_allowed %}
                                 <label class="item-checkbox-label">
                                     <input type="checkbox" value="1"
-                                            {% if c.max_count == 1 and c.min_count == 1 and c.items|length == 1 %}
-                                                checked="checked"
-                                                disabled="disabled"
-                                            {% elif item.initial %}
+                                            {% if c.max_count == 1 and c.min_count == 1 and c.items|length == 1 or item.initial %}
```

Comment:
I'd prefer not mixing "and" and "or" in templates as it is impossible to see in which order they are executed.

Can we maybe add a property/it to the object that is `c` here to collapse the "and" part?

Another way might be keeping the two conditions and just removing the disabled line. Some repetition is okay if it improves clarity.